kardia/rpc.go

```diff
@@ -520,6 +523,16 @@ func (ec *Client) Validators(ctx context.Context) ([]*types.Validator, error) {
 //	return result, nil
 //}
 
+//
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>commentFormatting: put a space between `//` and comment text (gocritic)